iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
since you are already refactoring, we can add some readme to this folder explaining about this sample.

Reply:
There isn't much to explain here, as in, it isn't a long running sample that follows a certain convention. I've marked out the individual steps in the sample itself, but if there is something additional you'd like me to add (or a step that isn't clear to you), let me know and I'll add it in.